.github/workflows/dependabot-pr.yml

```diff
@@ -13,13 +12,11 @@ jobs:
       - name: Dependabot metadata
         id: metadata
         uses: dependabot/fetch-metadata@v1
-        with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
       - name: Approve the PR & enable auto-merge
         if: ${{ steps.metadata.outputs.update-type == 'version-update:semver-patch' }}
         run: |
           gh pr review --approve "$PR_URL"
           gh pr merge --auto --merge "$PR_URL"
         env:
           PR_URL: ${{github.event.pull_request.html_url}}
-          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          GH_TOKEN: ${{secrets.SERVO_BOT_PR_TOKEN}}
```

Comment:
If this is a new token, would it be possible to rename it to SERVO_DEPENDABOT_TOKEN, so that we know what it is used for in the future when looking at project settings? We have a few other "bots" as well.

Reply:
Oh, I see now that this token is for a new, generically named servo-bot user. Do you have more plans for this bot as well?

Reply:
Yep, I'm trying to switch the nightly build release process to also use the new @servo-bot user, but that will use a different token, scoped to servo/servo-nightly-builds repo.

The token variable is named that way so that we can probably use it in other contexts where the bot can review, merge or otherwise interact with PRs in any way, not necessarily dependabot PRs, but I don't have any such specific use case in mind right now. Also, FYI, the fine-grained token itself is named 'Approve dependabot PRs' and the description of the token also documents the intended usage. This is just the name of the token variable in this specific repo. Let me know if you want to change this.

Reply:
That makes sense. If this token is used only to approve dependabot PRS for now let's just name it SERVO_DEPENDABOT_TOKEN until we expand the usage.